src/utils.js

```diff
@@ -176,6 +176,24 @@ const utils = {
 	},
 
 	/**
+	 * Returns the number of items return by the iterator.
+	 *
+	 *		utils.count( [ 1, 2, 3, 4, 5 ] ); // 5;
+	 *
+	 * @param {Iterable.<*>} iterator Any iterator.
+	 * @returns {Number} Number of items returned by that iterator.
+	 */
+	count( iterator ) {
```

Comment:
Since we moved utils to the separate package (https://github.com/ckeditor/ckeditor5-core/pull/251), we should move this change there too.